### ISSUE TITLE
Modularize JDOM

### ISSUE THREAD

**Issue by vatbub** [hunterhacker/jdom#169]
Knowing that modularization is a rather big thing, I wanted to suggest it for the future roadmap. 

Background: I am using JDOM in an android project and tried to exclude the jdom core as stated [in the wiki](https://github.com/hunterhacker/jdom/wiki/JDOM2-and-Android). Since I referenced the maven artifact as a dependency and thus don't have a libs folder in my repo where the jdom jar resides in, I can't just remove the jdom core jar because gradle handles the download of the jar. 
I can tell gradle to exclude certain parts of a jar (transitive dependencies), but only, if that transitive dependency is published as a seperate jar on maven central.

**TL;DR** I would like to exclude the jdom core from my build but am unable to do so. If jdom core was published as a separate artifact to maven central with jdom declaring a dependency on it, exclusion would be much easier.

**Comment by VISTALL** [comment]
Hello. I'm working at that issue (since i want migrate to java 9). 

You can found my source here https://github.com/consulo/jdom

Also this library will contains module-info for java 9 (but will be compiled to java 6)
